packages/core/src/arguments-resolver/page.ts

```diff
@@ -0,0 +1,57 @@
+import { isFunction, isPlainObject, isString } from '../validation'
+import { JSONObject, EventProperties, CoreOptions } from '../events'
+import { Callback } from '../events/interfaces'
+
```

Comment:
Basically unchanged, except for moving the arguments resolver to its own "page" file -- it's the only resolver we need to bring in (because of the overloads)